system/jlib/jlib.hpp

```diff
@@ -204,6 +204,8 @@ class PointerArrayOf : public PointerArray
     inline bool zap(TYPE * x)                   { return PointerArray::zap(x); }
 };
 
+enum SSLCfg { SSLNone = 0, SSLOnly, SSLFirst, LegacyFirst };
```

Comment:
Need a comment block somewhere describing how these options behave.
Probably also in documentation.

Does LegacyFirst mean insecure socket first ?

Would be more consistent if all these were prefixed with SSL